upstream-community-operators/rook/rook.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,515 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "ceph.rook.io/v1",
+          "kind": "CephCluster",
+          "metadata": {
+            "name": "my-rook-ceph",
+            "namespace": "rook"
```

Comment:
This namespace is `rook` and the other CR examples are in the `rook-ceph` namespace. What is the right default for operator hub? I would think `rook-ceph`

Reply:
OperatorHub does not make any assumptions about the namespaces settings in your CRs, so it's up to you.